app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,10 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
```

Comment:
I thought that we agreed to have this method in ```toolbar_builder.rb``` (also according to @martinpovolny comment https://github.com/ManageIQ/manageiq/pull/10916#issuecomment-244056369)